bundles/org.eclipse.mita.program/src/org/eclipse/mita/program/generator/StatementGenerator.xtend

```diff
@@ -618,6 +621,11 @@ class StatementGenerator {
 		return CodeFragment.EMPTY;
 	}
 	
+
+	dispatch def IGeneratorNode code(ReturnParameterDeclaration __) {
```

Comment:
Nit: even though you're not using the parameter here, for consistency's sake it might make sense to call this one `stmt` (they're called `stmt` everywhere else).